### DISCUSSION TITLE
possible bug when resizing window

### DISCUSSION THREAD

**Discussion by wnleao** [sevdokimov/log-viewer#21] [General]
I'm using firefox 68.10.0esr (64-bit) at work. I'm able to load the logfile, but when I start resizing the window  (expanding its width) the loglines start to fit in a single (no breaks) then the log panel shrinks its height (the end marks go up). After that, when I try to resize the window back to its original width the log panel do not expand its height back to what it was and the content breaks in the sense that the log panel now occupies half the full window. I don't if this is a known problem or its due to my old firefox version. Sorry but I do not have other browser to test this behavior.

**Comment by sevdokimov** [comment]
I cannot reproduce the problem. On my FF 78.0.2 everything works fine.

**Reply by wnleao** [reply]
I'll try and test it with a newer version. My firefox at work is not that great. thank you!
